.github/actions/install-pyroscope/action.yaml

```
@@ -0,0 +1,32 @@
+name: InstallGrafana
+description: 'Installs grafana'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+    - name: download pyroscope
+      shell: bash
+      run: | 
+        curl -fL https://github.com/grafana/pyroscope/releases/download/v1.1.5/profilecli_1.1.5_linux_amd64.tar.gz | tar xz profilecli
```

```suggestion
        curl -fL https://github.com/grafana/pyroscope/releases/download/v1.1.5/profilecli_1.1.5_linux_amd64.tar.gz | tar xz pyroscope
```

nit: consider renaming

---

The binary itself when we curl it is called profilecli. renaming it would be a bit redundant too

---

Oh yeah lol. Dumb comment. Ignore me 😆